scripts/core/editor3/directive.js

```diff
@@ -111,6 +115,14 @@ class Editor3Directive {
 
         const store = createStore(this);
 
+        $scope.$watch('vm.value', (newValue, oldValue) => {
```

Comment:
To detect if the value sent to editor3 wrapper directive has been changed and then to update the editor3 state.